packages/http-cors/index.js

```diff
@@ -1,10 +1,19 @@
 import { normalizeHttpResponse } from '@middy/util'
 
 const getOrigin = (incomingOrigin, options = {}) => {
+  const wildcardMatch = (origin) => {
+    const escapedString = origin.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')
```

Comment:
MDM recommended regular expression escaping.
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions#escaping

FYI - There is an ECMAScript [proposal](https://github.com/tc39/proposal-regex-escaping) to add such a function to RegExp.